### PR TITLE
35 create infra model

### LINKED ISSUE #35: Create Infra model
Create an infra model file in `models` folder

added model with appropriate key values and conditions.
Models should be added in the model folder. 
file name should be `infra.js`

### PR DESCRIPTION
**Pull request by ANKIT638-ux**
fixes #35

### CODE REVIEW

File: models/infra.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const subjectContent = new connector.model("subject content", subjectContentSchema);
+module.exports = {};
```
Comment (TejasNair9977) [inline models/infra.js:2]: can you remove this? this is probably from another file. please dont delete your branch, just remove this, commit it, and then push your work, on your branch.

**Review by TejasNair9977 — CHANGES_REQUESTED** [review]
please make the changes.

File: models/infra.js (hunk 1 above)
Comment (ANKIT638-ux) [reply models/infra.js:2]: ok

**Review by Hitansh159 — CHANGES_REQUESTED** [review]
Make the following changes

File: models/infra.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const { connector } = require("#models/databaseUtil");
```
Comment (Hitansh159) [inline models/infra.js:1]: do not use require. use import

**Review by Hitansh159 — APPROVED** [review]

**Review by TejasNair9977 — APPROVED** [review]
LGTM

**Pull request merged**